test/functional/api/v1/webauthn_verifications_controller_test.rb

```diff
@@ -0,0 +1,70 @@
+require "test_helper"
+
+class Api::V1::WebauthnVerificationsControllerTest < ActionController::TestCase
+  should "route new paths to new controller" do
+    route = { controller: "api/v1/webauthn_verifications", action: "create" }
+    assert_recognizes(route, { path: "/api/v1/webauthn_verification", method: :post })
+  end
+
+  def authorize_with(str)
+    @request.env["HTTP_AUTHORIZATION"] = "Basic #{Base64.encode64(str)}"
+  end
+
+  context "on POST to create" do
+    context "with no credentials" do
+      setup { post :create }
+      should_deny_access
+    end
+
+    context "with invalid credentials" do
+      setup do
+        @user = create(:user)
+        create(:webauthn_credential, user: @user)
+        authorize_with("bad\0:creds")
+        post :create
+      end
+
+      should "deny access" do
+        assert_response 401
+        assert_match "HTTP Basic: Access denied.", @response.body
+      end
+    end
+
+    context "user has enabled webauthn" do
+      setup do
+        @user = create(:user)
+        create(:webauthn_credential, user: @user)
+        authorize_with("#{@user.email}:#{@user.password}")
+        post :create, format: :yaml
+      end
+
+      should respond_with :success
+
+      should "return Webauthn verification URL with path token" do
+        response = YAML.load(@response.body)
```

Comment:
Unlike the API key controller, we assume for now that YAML is the only supported format. We can add JSON later if necessary, but at the moment this endpoint is only intended for the use of the CLI.